utils/segmaker.py

```diff
@@ -39,31 +39,26 @@ def compile(self, bitfilter=None):
 
```

Comment:
Yes. Also not sure how this relates to this PR. It would be helpful to add comments to this file with or without the PR, and looking at the concrete changes I don't see how the PR makes the situation any worse. It's all just changes like `segdata["baseaddr"][1]` -> `tiledata["offset"]` to accommodate the new tilegrid format.